With a recent urllib, test_instance fails:

koji> ======================================================================
koji> ERROR: test_instance (tests.test_lib.test_auth.TestAuthSession.test_instance)
koji> Simple kojihub.auth.Session instance
koji> ----------------------------------------------------------------------
koji> Traceback (most recent call last):
koji>   File "/build/source/tests/test_lib/test_auth.py", line 55, in test_instance
koji>     kojihub.auth.Session()
koji>   File "/build/source/kojihub/auth.py", line 100, in __init__
koji>     args = urllib.parse.parse_qs(args, strict_parsing=True)
koji>            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
koji>   File "/nix/store/glfr70gi7hfaj50mwj2431p8bg60fhqw-python3-3.11.9/lib/python3.11/urllib/parse.py", line 718, in parse_qs
koji>     pairs = parse_qsl(qs, keep_blank_values, strict_parsing,
koji>             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
koji>   File "/nix/store/glfr70gi7hfaj50mwj2431p8bg60fhqw-python3-3.11.9/lib/python3.11/urllib/parse.py", line 772, in parse_qsl
koji>     qs = bytes(memoryview(qs))
koji>                ^^^^^^^^^^^^^^
koji> TypeError: memoryview: a bytes-like object is required, not 'MagicMock'
koji> ----------------------------------------------------------------------

Skip the test for now.

--- a/tests/test_lib/test_auth.py	1980-01-02 00:00:00.000000000 +0000
+++ b/tests/test_lib/test_auth.py	2024-05-01 15:51:53.001264568 +0100
@@ -44,6 +44,7 @@
         # start with "assert"
         self.context.session.assertLogin = mock.MagicMock()
 
+    @unittest.skip("urllib.parse.parse_qsl calls bytes() on a memoryview which is a mock and crashes")
     def test_instance(self):
         """Simple kojihub.auth.Session instance"""
         self.context.opts = {
